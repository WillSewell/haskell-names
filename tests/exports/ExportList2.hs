module ExportList2 (b, c) where

data A
  = A { b :: A }
  | B { b :: A, d :: A }

c = b
