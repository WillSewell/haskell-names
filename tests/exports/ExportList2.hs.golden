Symbols (
  fromList
    [ SymValue
        { sv_origName = OrigName
                          { origPackage = Nothing
                          , origGName = GName
                                          { gModule = "ExportList2"
                                          , gName = "c"
                                          }
                          }
        , sv_fixity = Nothing
        }
    , SymSelector
        { sv_origName = OrigName
                          { origPackage = Nothing
                          , origGName = GName
                                          { gModule = "ExportList2"
                                          , gName = "b"
                                          }
                          }
        , sv_fixity = Nothing
        , sv_typeName = OrigName
                          { origPackage = Nothing
                          , origGName = GName
                                          { gModule = "ExportList2"
                                          , gName = "A"
                                          }
                          }
        , sv_constructors = [ OrigName
                                { origPackage = Nothing
                                , origGName = GName
                                                { gModule = "ExportList2"
                                                , gName = "A"
                                                }
                                }
                            , OrigName
                                { origPackage = Nothing
                                , origGName = GName
                                                { gModule = "ExportList2"
                                                , gName = "B"
                                                }
                                }
                            ]
        }
    ] ) (
  fromList [] )